docs/technical/installation_controller.md

```diff
@@ -0,0 +1,38 @@
+# Installation Controller
+
+The reconciliation flow of the installation controller is described in detail [here](./reconciliation_flow.md).
+It executes one of the following main operations:
+
+- a deletion,
+- a [standard reconcile](#standard-reconcile),
+- a [force reconcile](#force-reconcile),
```

Comment:
the path "imports satisfied? -yes-> return error seems odd and doesnt fit the description in reconciliation_flow.md, does it?

Reply:
You are right, I mixed up "yes" and "no". I have updated the image.